darwin/future/meta/objects/workflow.py

```diff
@@ -0,0 +1,23 @@
+from typing import List, Optional
+
+from darwin.client import Client
+from darwin.future.data_objects.workflow import Workflow
+from darwin.future.meta.objects.base import MetaBase
+
+
+class WorkflowMeta(MetaBase[Workflow]):
```

Comment:
I think we should swap this ordering around, people are going to be using meta objects more than they are Core, so think it makes more sense to have WorfklowCore and the Meta is just Workflow. Doesn't have to be done now though but I might do it at some point.

Reply:
Ah yeah, that makes sense.  Feels like a all of them at once job, otherwise it doesn't make much sense - they fall out of sync with each other.